index.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+<link rel="stylesheet" href="https://cdn.linearicons.com/free/1.0.0/icon-font.min.css">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>Login</title>
+
+<!--CSS-->
+<link rel="stylesheet" href="css/main.css">
+<link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i%7CPlayfair+Display:400,400i,700,700i,900,900i" rel="stylesheet">
+<link rel="stylesheet" href="https://cdn.linearicons.com/free/1.0.0/icon-font.min.css">
+<link rel="stylesheet" href="assets/pe-icon-7-stroke/css/pe-icon-7-stroke.css">
+
+</head>
+
+<body>
+
+	<div class="loginCon"> 
+		<div class="row">
+			<!--LeftSide-->
+			<div class="left-log column-large-6 column-medium-5 show-for-large align-center cover-content">
+                <div class="bgimg"></div>
+			</div>
+ 
+            <!--RightSide-->
+            <div class="right-log column-large-6 column-medium-5 column-small-8 align-center cover-content">
+                <div class="whiteLogo show-for-small" alt="white">
+                <img src="images/LogoWhite.svg" alt="logowhite">
+            </div>
+                <div class="vertical-logo align-center show-for-medium">
+                     <img src="images/logo.svg" alt="logo">
+                </div>
+            <div class="loginForm">
+            <form name="login-form" id="login-form" class="contact-form js-animation" method="pos"> 
+            	   <p>Welcome back! Please login to your account.</p>
+
+               <!--  <div class="form-field">
+                    <input id="username" class="input-field text-left" type="text" required>
+                    <span class="lnr lnr-user"></span>
+                    <label class="label" for="username">UserName*</label>
+                </div>
+                <div class="form-field">
+                    <input id="pasword" class="input-field text-left" type="password" required>
+                     <span class="lnr lnr-lock"></span>
+                    <label class="label" for="password">Password*</label>
+                </div> -->
+
+                <div class="form-field-check row text-center align-center">
+                        <div class="checkbox column-large-6">
+                         <input type="checkbox" name="RememberMe" value="Remember"> Remember Me
```

Comment:
This is showing on mobile, but the design file doesn't have this on mobile.

Reply:
fixed